bitcoin/src/taproot.rs

```diff
@@ -544,15 +556,14 @@ pub struct NodeInfo {
 impl NodeInfo {
     /// Creates a new [`NodeInfo`] with omitted/hidden info.
     pub fn new_hidden_node(hash: sha256::Hash) -> Self {
```

Comment:
This should accept `TapNodeHash`. Or am I missing something? (Only this one is blocking.)

Reply:
Duh, I swear all instances of sha256::Hash in this PR, but looks like I screwed up in rebasing somewhere